Automattic/woocommerce-payments
# Use dedicated data attributes for E2E tests

Fixes #3425 

#### Changes proposed in this Pull Request
Some E2E tests currently use nested CSS properties for grabbing elements that might change, introducing flakiness in the tests. This adds custom `data-testid` attributes to elements and use them to select elements in E2E tests.








#### Testing instructions





* Go to Actions->WCPay E2E tests->Run workflow.
* Select this branch: `add/e2e-test-specific-identifiers` and run the workflow.
* All refactored tests should still pass, [reference job](https://github.com/Automattic/woocommerce-payments/actions/runs/1794296607)

-------------------


**Post merge**
